authentication/signUpPage/signup.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- <link rel="stylesheet" href="./" /> -->
+    <link rel="stylesheet" href="./signup.css" />
+    <link rel="stylesheet" href="./rootsignup.css" />
+    <title>SignUp | User</title>
+  </head>
+  <body>
+    <header class="flex-sa">
+      <h1 class="arima_madurai vilvah-green">
+        <a href="/index.html" class="arima_madurai vilvah-green"> Vilvah</a>
+      </h1>
+
+      <input
+        class="input-items arima_madurai_100"
+        type="text"
+        placeholder="Search for products"
+      />
+      <ul class="arima_madurai_100">
+        <li><a href="#">Wishlist</a></li>
+        <li><a href="#">Cart</a></li>
+        <li><a href="#">Profile</a></li>
+        <li><a href="#">Login</a></li>
+      </ul>
+    </header>
+
+    <main class="flex-center">
+      <div class="user-login">
```

Comment:
Instead of wrapping inside a `div`, you can use `form` tag here.

Reply:
ok thanks for poiniting it out